api/database/factories/UserFactory.php

```diff
@@ -69,23 +74,24 @@ public function definition()
             'looking_for_english' => $lookingEnglish,
             'looking_for_french' => $lookingFrench,
             'looking_for_bilingual' => $lookingBilingual,
-            'bilingual_evaluation' => $hasBeenEvaluated ? $this->faker->randomElement([
-                BilingualEvaluation::COMPLETED_ENGLISH->name,
-                BilingualEvaluation::COMPLETED_FRENCH->name,
-            ]) : BilingualEvaluation::NOT_COMPLETED->name,
-
-            'comprehension_level' => $hasBeenEvaluated ?
+            'first_official_language' => $lookingBilingual ? $this->faker->randomElement([
```

Comment:
I think this seeded value might be throwing the system off, because the enum has different values than the names.  I get an error when I try to log in as a seeded user.
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/f1130138-aca6-4925-8a38-415fb22199a2)

Reply:
If I change first_official_language to lowercase manually, it fixes it.

Reply:
Fixed by [4a8d201](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9766/commits/4a8d2010d6f39c6a7d5ea0ec5932436728f8ddb5)

Reply:
I'm not sure that just `strtoupper` is the best approach here.  Right now the enum name _is_ just the uppercase of the enum value but it won't necessarily _always_ be that.  It would probably better to do [an enum parse](https://www.php.net/manual/en/language.enumerations.backed.php).
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/d43e178c-14eb-48bd-9e2c-d23cb9e97819)

Reply:
Oh, you're right, this pattern is already used in a few places.  Let's leave as-is for now then.

Reply:
We discussed this today and decided to keep this pattern.

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/22059495/da387594-2ae8-4e2a-b840-6d5c00465dd1)